### PR TITLE
Adding German locale.

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

File: bepdf/locales/de.catkeys [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+Add help text annotation	PDFWindow		Anmerkung:  Hilfetext
+Custom zoom factor (%d%%)	PDFWindow		Eigener Zoomfaktor (%d%%)
+Open in new window…	PDFWindow		In neuem Fester öffnen…
+Add highlight annotation	PDFWindow		Anmerkung: Hervorheben
+Page: %d	FindTextWindow		Seite: %d
+Show all fonts	FileInfoWindow		Alle Schriften zeigen
+en
+en
+en
+en
```
Comment (waddlesplash) [inline bepdf/locales/de.catkeys]: ????

File: bepdf/Makefile [hunk 2]
```diff
@@ -128,7 +128,7 @@ OPTIMIZE := SOME
 # 	will recreate only the "locales/en.catkeys" file. Use it as a template
 # 	for creating catkeys for other languages. All localization files must be
 # 	placed in the "locales" subdirectory.
-LOCALES =
+LOCALES = en de
```
Comment (waddlesplash) [inline bepdf/Makefile]: Is there any point to the "en" locale, since BePDF is in English anyway?

Comment (humdingerb) [inline bepdf/Makefile]: Yes. Once you declare one locale in the Makefile you need to include the default "en" as well or it won't be compiled in.

File: bepdf/locales/de.catkeys (hunk 1 above)
Comment (humdingerb) [inline bepdf/locales/de.catkeys]: Fixed that with the updated PR. That's some strange bug in the CatKeysEditor...

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**